app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -1,5 +1,85 @@
+/* global nodeListForEach */
+//  = require ../vendor/polyfills/common.js
```

Comment:
In an earlier commit the polyfill for `indexOf` was removed, but code containing this is now restored in this commit. Can this polyfill be added back in as well?

Reply:
It's been removed as it's handled at the `govuk-frontend` side